crates/api_crud/src/user/create.rs

```diff
@@ -174,14 +177,9 @@ pub async fn register(
   if !local_site.site_setup
     || (!require_registration_application && !local_site.require_email_verification)
   {
-    login_response.jwt = Some(
-      Claims::jwt(
-        inserted_local_user.id.0,
-        &context.secret().jwt_secret,
-        &context.settings().hostname,
-      )?
-      .into(),
-    );
+    let jwt = Claims::generate(inserted_local_user.id, req, &context).await?;
+    res.cookie(create_login_cookie(jwt.clone()));
+    login_response.jwt = Some(jwt);
```

Comment:
Same question here. Why are we both setting a cookie and returning the same cookie as a http response?

Reply:
Some clients may not support cookies, or may want to login via auth header for some reason. Setting the cookie here is not necessary, but it can be more convenient for clients as they dont have to worry about auth then.